package.json

```
@@ -38,7 +38,8 @@
     "precypress": "node ./cypress-install.js",
     "cypress": "cypress",
     "cypress:dev:run": "npm run cypress -- run",
```

```suggestion
    "cypress:dev:run": "npm run cypress -- run --spec \"cypress/integration/main-tests/**/*\"",
    "cypress:dev:run:full": "npm run cypress -- run --spec \"cypress/integration/**/*\"",
```
`cypress open` doesn't let you filter via the spec argument (though `run` does).  Instead you can use the in-built search function to achieve the same thing.

The quotes around the glob should make this work on Windows. Also, Cypress warns you if they're missing.